sklearn/mixture/dpgmm.py

```diff
@@ -106,6 +116,8 @@ def _bound_state_log_lik(X, initial_bound, precs, means, covariance_type):
     return bound
 
 
+@deprecated("The class DPGMM is deprecated in 0.18 and "
+            "will be removed in 0.20. This class no longer works.")
```

Comment:
I'm not sure about "this class no longer works". I think we should say explicitly that it will provide bad results. Maybe ``This class is broken and may yield bad results"?